app/services/live_config/store.py

```
@@ -235,6 +235,41 @@ async def get_config(key: str, default_value: Any, return_type: type = str) -> A
+        # Load existing flags
+        raw = await client.get(FEATURE_FLAGS_KEY)
+        all_flags = json.loads(raw) if raw else {}
+
+        # Update the specific key (normalize to match DevCycle format)
+        normalized_key = normalize_key(key)
+        all_flags[normalized_key] = value
+
+        # Write back
+        await client.set(FEATURE_FLAGS_KEY, json.dumps(all_flags))
+        logger.info(f"Runtime config updated: {normalized_key} = {value}")
+        return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)